### PR TITLE
Reran `make mappings` for gard omim and ordo

### LINKED ISSUE #353: `lexmatch` is missing umapped_ordo tables
For some reason the ordo lexmatch table is missing.

https://github.com/monarch-initiative/mondo-ingest/blob/main/src/ontology/slurp/ordo.tsv#L60

- perrault syndrome type 1 should have been a lexmatch (same exact synonym)

@hrshdhgd high priority issue

Is it somehow possible to create an empty file if there are no matches, and not let it disappear?

I assume this is some kind of prefix mismatch again.

### PR DESCRIPTION
**Pull request by hrshdhgd**
Fixes #353 

Steps taken:

 - `make ../mappings/gard.sssom.tsv`
 - `make ../mappings/omim.sssom.tsv`
 - `make ../mappings/ordo.sssom.tsv`
 - `make matches`
 - added missing `if` condition in `lexmatch-sssom-compare.py`

### CODE REVIEW

**Comment by matentzn** [comment]
Can you explain again what exactly happened?

**Comment by hrshdhgd** [comment]
I cannot yet ... still testing hypotheses ...

File: src/scripts/lexmatch-sssom-compare.py [hunk 1]
```diff
@@ -176,6 +176,8 @@ def extract_unmapped_matches(input: str, matches: TextIO, output_dir: str, summa
         ont2 = ont.upper()
         if ont == "omim":
             ont2 = "|".join((["OMIM", "OMIMPS"]))
+        elif ont == "ordo":
+            ont2 = "|".join((["ORDO", "Orphanet"]))
```
Comment (hrshdhgd) [inline src/scripts/lexmatch-sssom-compare.py:180]: @matentzn : this missing condition was the culprit of not catching `Orphaned:XXXX` terms. As per the incorrect prefix, my guess is someone ran the script without providing `-c metadata/mondo.sssom.config.yml` and pushed it to GitHub. I don't have any other explanation.

File: src/ontology/lexmatch/mondo-only/unmapped_ordo_mondo.tsv
Comment (hrshdhgd) [inline src/ontology/lexmatch/mondo-only/unmapped_ordo_mondo.tsv:1]: ordo is back!

File: src/ontology/lexmatch/split-mapping-set/mondo_exactmatch_orphanet.tsv
Comment (hrshdhgd) [inline src/ontology/lexmatch/split-mapping-set/mondo_exactmatch_orphanet.tsv:1]: Orphaned now exists!

File: src/ontology/lexmatch/unmapped_ordo_lex_exact.tsv
Comment (hrshdhgd) [inline src/ontology/lexmatch/unmapped_ordo_lex_exact.tsv:1]: Hence this update!

File: src/scripts/lexmatch-sssom-compare.py (hunk 1 above)
Comment (matentzn) [reply src/scripts/lexmatch-sssom-compare.py:180]: Ok, thank you! Perfect detective work!

File: src/ontology/lexmatch/mondo-only/unmapped_ordo_mondo.tsv
Comment (matentzn) [reply src/ontology/lexmatch/mondo-only/unmapped_ordo_mondo.tsv:1]: 🌮 🎉

**Review by matentzn — APPROVED** [review]
Fantastic work @hrshdhgd really appreciated, very important!!

**Review by matentzn — APPROVED** [review]

**Pull request merged**